Library/Homebrew/commands.rb

```diff
@@ -143,4 +143,17 @@ def find_commands(path)
             .select(&:file?)
             .sort
   end
+
+  def rebuild_internal_commands_completion_list(path)
+    cmds = internal_commands + internal_developer_commands + internal_commands_aliases
+    File.open(path, "wb") do |f|
+      f.write(cmds.sort.join("\n"))
+    end
+  end
+
+  def rebuild_commands_completion_list(path)
+    File.open(path, "wb") do |f|
+      f.write(commands(aliases: true).sort.join("\n") + "\n")
+    end
```

Comment:
```suggestion
    Pathname(path).write(commands(aliases: true).sort.join("\n") + "\n")
```

Reply:
Done, although had to add a check before to remove existing file, since `Pathname.write` will not overwrite.